lib/swagger-client.js

```diff
@@ -1460,22 +1460,8 @@ Operation.prototype.encodeQueryParam = function(arg) {
   return encodeURIComponent(arg);
 };
 
-/**
- * TODO revisit, might not want to leave '/'
- **/
 Operation.prototype.encodePathParam = function(pathParam) {
-  var encParts, part, parts, i, len;
```

Comment:
That's true. 

```
encodeURIComponent('a/b') === "a%2Fb"
'a/b'.split('/').map(encodeURIComponent).join('/') === 'a/b'
```
if we decided to have an option to allow slash(`/`), we would need to skip encoding it. Your code skip encoding slash. My suggestion is to make that code shorter and less complex.

Based on @fehguy's comment we need to decide if we want to skip slash in encoding and if the answer to that is yes, we should make this behavior configurable.